.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+<!--
+DO NOT FILE ISSUES FOR GENERAL SUPPORT QUESTIONS.
```

Comment:
Same comment here as in `ISSUE_TEMPLATE`. I'm fine with answering general support questions here for now.